### ISSUE TITLE
how to disable web security!!

### ISSUE THREAD

**Issue by zeko2010ss** [scraperlab/browserext#8]
I wonder how can I how to disable web security!!

**Comment by scraperlab** [comment]
What exactly do you mean?

**Comment by zeko2010ss** [comment]
In chrome we use --disable-web-security to allow disable same origin policy and in Phantomjs we can use --web-security=false

**Comment by scraperlab** [comment]
I think without a patching QtWebKit, this can not be done. But browserext uses original library. As far as I know, the phantomjs uses a modified QtWebKit. But I can be wrong.

**Comment by zeko2010ss** [comment]
so there is no way to add this method in browserext later and control iframes also

**Comment by scraperlab** [comment]
yes

**Comment by zeko2010ss** [comment]
I'm waiting for this thanks sir

**Comment by scraperlab** [comment]
There is nothing to wait from browserext, with the original Qt library it can not be done.

**Comment by scraperlab** [comment]
In view of the fact that the official QtWebKit binaries is no longer included in the new versions of the Qt (since 5.6), perhaps we will switch to an unofficial version, and then we will be able to realize your request.

But when it is still unknown.

**Comment by zeko2010ss** [comment]
oh I see thanks

**Comment by zeko2010ss** [comment]
so how we could control iframes!

**Comment by scraperlab** [comment]
No way.
Why do you not want to use Phantomjs or Selenium? Their capabilities are much wider than the browserext.

**Comment by zeko2010ss** [comment]
I see that browserext very stable and powerful Phantomjs or Selenium make much load on server but browserext need more options like control frames & disable load images and web security so it will be better than Ghost.py

**Comment by scraperlab** [comment]
But the Phantomjs also uses QtWebKit, so the server load should be the same as the browserext.

**Comment by scraperlab** [comment]
Disabling load images is supported. See setImageLoading method in PhpBrowser class.

**Comment by zeko2010ss** [comment]
yes you are right but keep in mind Phantomjs work with js so we need to use Selenium with Phantomjs  driver so we can use PHP or Python but browserext work without Selenium  Server

**Comment by zeko2010ss** [comment]
I love browserext and Ghost.py

**Comment by scraperlab** [comment]
I'm glad to hear that.
In the near future I will think over the issue of switching to an unofficial QtWebKit binaries, like Phantomjs.
Just wait...

**Comment by zeko2010ss** [comment]
I'm waiting for you thanks

**Comment by zeko2010ss** [comment]
method jsexec not work browser exit

**Comment by scraperlab** [comment]
Maybe your javascript has an error.
I try this code:
```php
$browser = new PhpBrowser();
$browser->load('http://www.google.com');
echo $browser->title();
$a = $browser->elements('//a[1]');
$a[0]->jsexec('alert(123);');
echo $browser->console();
```
It works.
Show your js code here.
